src/demo/DeckGLMapDemo.js

```diff
@@ -71,6 +71,7 @@ const DeckGLMapDemo = () => {
 
     const [jsonData, setJsonData] = React.useState(null);
     const [colormaps, setColormaps] = React.useState([]);
+    const [showCoords, setShowCoords] = React.useState(true);
```

Comment:
Looking at the usage here, it might be easier to use useMemo()?
const showCoords = useMemo(()=>typeof jsonData.showCoords === "undefined"
                ? true
                : jsonData.showCoords,[jsonData])
Btw, what does typeof jsonData.showCoords === "undefined" do? Is it for checking if a field is present in the json?

Reply:
Yes, if the field doesn't exist in the json, showCoords should be true.